src/main/java/de/hysky/skyblocker/skyblock/item/PriceInfoTooltip.java

```diff
@@ -353,7 +353,7 @@ private static Text getMotesMessage(int price, int count) {
 
     // If these options is true beforehand, the client will get first data of these options while loading.
     // After then, it will only fetch the data if it is on Skyblock.
-    public static int minute = -1;
+    public static volatile int minute = -1;
```

Comment:
I'm getting warnings for the non-atomic ++ operations on this field. This field doesn't need to be volatile since it's only used inside the lambda?
```suggestion
    public static int minute = -1;
```